### PR TITLE
[CRIMAPP-376] Display application type across review

### PR DESCRIPTION
**Pull request by hiboabd**
## Description of change
Adds column displaying application type to the open/closed application, search and assigned application tables 
Adds pse feature flag 

## Link to relevant ticket
[CRIMAPP-376](https://dsdmoj.atlassian.net/browse/CRIMAPP-376)

### After changes:

Screenshot 2024-01-25 at 17 42 43
Screenshot 2024-01-25 at 17 42 36
Screenshot 2024-01-25 at 17 42 28
Screenshot 2024-01-25 at 17 42 18

## How to manually test the feature

Build review from this branch and then navigate to the following pages and check the table displayed has a column for application type, displaying the application type expected

- Assigned applications page (Your list)
- Open applications page
- Closed applications page
- Application search results page

Note: Make sure you have built datastore and crime apply from main

- Submit a standard application in apply and review the application in review (as can only submit pse for applications that are complete)
- Then go back to apply and find the application in the submitted tab. Click on the application and click upload post submission evidence
- Submit your pse application and then go to review and check the tables display the correct application type value

Also turn off the pse feature flag and test the columns are not shown

[CRIMAPP-376]: https://dsdmoj.atlassian.net/browse/CRIMAPP-376?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/casework/application_searches/_search_results_table_body.html.erb
Comment (hiboabd) [inline app/views/casework/application_searches/_search_results_table_body.html.erb:16]: In the search table, case type is displayed but this is not present for pse applications, so have left blank for now following the existing behaviour for other columns

**Review by ripan — APPROVED** [review]
LGTM 👀 . Please wait for 1 more approval before merging 😄

**Review by timpeat — APPROVED** [review]

**Pull request merged**
